Aptoide/aptoide-client-v8
# [ASV-1767] Chips cancel button

**What does this PR do?**

   This PR aims at only showing the cancel button when a chip is selected. Also changed background of editorial cards in editorial view.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] HomeContainerFragment.java

**How should this be manually tested?**

  Open Aptoide in both dark theme and non dark theme and see that the cancel button only shows up if the chip is selected.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1767](https://aptoide.atlassian.net/browse/ASV-1767)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
